controllers/trackController.js

```diff
@@ -1,131 +1,117 @@
-var bodyParser=require('body-parser');
-var mongoose= require('mongoose');
-mongoose.connect('mongodb://username:password@databaseserver:port/databasename',{ useNewUrlParser: true});
+const bodyParser = require('body-parser');
+const dotenv = require('dotenv');
+dotenv.config();
+const mongoose = require('mongoose');
+mongoose.connect('mongodb+srv://' + process.env.MONGOUSER + ':' + process.env.MONGOPASSWORD + '@cluster0.jlrkv.mongodb.net/njmusicstore?retryWrites=true&w=majority', { useNewUrlParser: true });
 /*Replace the above connection string with the actual connection string of your MongoDB database*/
-var trackSchema= new mongoose.Schema({    
-    trackName:String,
-    artistName:String,
-    albumName:String,
-    albumYear:String,
-    albumGenre:String,  
-    trackPrice:String
- });
-var Track=mongoose.model('Track',trackSchema);
-var userCartSchema = new mongoose.Schema({
-    username:String,
-    trackid:String,
-    trackname:String,
-    quantity:String,
-    unitprice:String
+const trackSchema = new mongoose.Schema({

```

Comment:
Schemas and/or db functions are part of the `model` layer  of the MVC pattern, consider refactoring these to  a separate `models` structure either now or in the future :)

Reply:
Will do. Thanks